theano/sandbox/cuda/basic_ops.py

```diff
@@ -3340,6 +3319,75 @@ def c_code(self, node, name, inputs, out_, sub):
                 %(fail)s;
             }
         }
+        """ % locals()
+        return str
+
+    def infer_shape(self, node, input_shapes):
+        return [node.inputs]
+
+    def grad(self, inputs, grads):
+        gout, = grads
+        return [None for i in inputs]
```

Comment:
Just delete this method, `None` has not been a valid grad for a long time and nobody noticed.